components/org.wso2.carbon.identity.scim2.common/src/main/java/org/wso2/carbon/identity/scim2/common/impl/SCIMUserManager.java

```diff
@@ -1515,6 +1515,35 @@ private boolean isDomainNameEmbeddedInAttributeValue(String filterOperation, Str
         return false;
     }
 
+    private Set<org.wso2.carbon.user.core.common.User> filterUsersByGroup(Node node, int offset, int limit,
+                                                                          String domainName) throws CharonException {
+
+        // Set filter values.
+        String attributeName = ((ExpressionNode) node).getAttributeValue();
+        String filterOperation = ((ExpressionNode) node).getOperation();
+        String attributeValue = ((ExpressionNode) node).getValue();
+
+        // If there is a domain, append the domain with the domain separator in front of the new attribute value if
+        // domain separator is not found in the attribute value.
```

Comment:
The multi-line comments should follow the following format.
```
/*
 *
 *
 */
```

Reply:
Resolved with https://github.com/wso2-extensions/identity-inbound-provisioning-scim2/pull/395